### ISSUE TITLE
fail to change token claims for logged in user - WEB-player SDK

### ISSUE THREAD

**Issue by kapekost** [spotify/web-api#1109]
Issue found on February 19th 2015.

#### Endpoint(s):
* `GET /v1/v1/me/player`

#### Scope(s):
* Various (application under development)

#### Steps to reproduce:
1. Request token with scopes A and B
2. Use the Web SDK and login
3. Change the token scopes for the next use (add scope C)
4. Reload the browser and navigate all the way through the login (automatically logged in)

#### Expected behaviour:
We should be able to perform more action with the new scope with no error

#### Actual behaviour:
( maybe related to #520 and more relevant to the web player SDK )
The new token authenticates correctly, but the actions requiring scope C are not authorised, unless we manually clear the cache form the application tab (hard reload won't work either)

**Comment by kapekost** [comment]
hey, sorry for the late reply,
I think this is the case I am facing, redirecting to /authorize with the token that has more permissions, and those are getting ignored. 

I will update the details about that part and notify you when I get back to the project, 

Thank you

**Comment by kapekost** [comment]
This is how it happened:
a) the call to `/authorize` with the new scopes gives me the code
b) then in the callback function gets a token `https://accounts.spotify.com/api/token` and passes that to the client SDK to continue with the player component. 

at the first run I had a few scopes .. and I got a permission error from the client. 
then I reloaded my service while I fixed the scope

hard reloaded the client, and also made sure I am passing to the SDK always the token from my service. 

When I tried after reloading everything, I got the same error. 
it worked only when I cleared the browser cache, manually deleted absolutely everything (local cache and any sort of memory)
